lib/logstorage/syslog_parser.go

```diff
@@ -132,7 +132,11 @@ func (p *SyslogParser) Parse(s string) {
 
 	bufLen = len(p.buf)
 	p.buf = marshalUint64String(p.buf, severity)
-	p.addField("severity", bytesutil.ToUnsafeString(p.buf[bufLen:]))
```

Comment:
In my opinion, we should preserve the severity (and priority, in the case of the journal) and include it as a level. These fields are part of the protocol/spec, and there may be tools or queries that depend on their presence.